src/interface/interface.json

```diff
@@ -186,6 +186,85 @@
+                                    "desc": "closes the profile"
+                                },
+                                "setActive": {
+                                    "desc": "Pass an instance of either Account or Project",
+                                    "params": [
+                                        "{Account or Project} - The account or project currently chosen"
+                                    ]
+                                },
+                                "onClickOutside": "HIG.abstract.EventObject",
+                                "onClick": "HIG.abstract.EventObject"
+                            },
+                            "partials": {
+                                "Account": {
+                                    "methods": {
+                                        "setLabel": {
+                                            "desc": "sets the label",
+                                            "params": [
+                                                "{String} label - new label"
+                                            ]
+                                        },
+                                        "setImage": {},
+                                        "activate": {
+                                            "desc": "activates the account"
+                                        },
+                                        "deactivate": {
+                                            "desc": "deactivates the account"
+                                        },
+                                        "onClick": "HIG.abstract.EventObject"
+                                    },
+                                    "defaults": {
+                                        "image": "{String} url src of the image"
+                                    }
+                                },
+                                "Project": {
+                                    "methods": {
+                                        "setLabel": {
+                                            "desc": "sets the label",
+                                            "params": [
+                                                "{String} label - new label"
+                                            ]
+                                        },
+                                        "setImage": {},
+                                        "activate": {
+                                            "desc": "activates the project"
+                                        },
+                                        "deactivate": {
+                                            "desc": "deactivates the project"
+                                        },
+                                        "onClick": "HIG.abstract.EventObject"
+                                    },
+                                    "defaults": {
+                                        "image": "{String} url src of the image"
```

Comment:
I left out default label because I couldn't imagine a useful default.

Also, I was thinking of removing the default image because it looks like the desired design is to show the initials instead.